06.bowling_object/frame.rb

```diff
@@ -8,8 +8,9 @@ def initialize(frame_number)
   end
 
   def record_shot(shot_score)
-    @bonus_scores << shot_score if need_bonus?
-    @shot_scores << shot_score unless finished?
+    score = (shot_score == 'X') ? 10 : shot_score.to_i
```

Comment:
```suggestion
    score = shot_score == 'X' ? 10 : shot_score.to_i
```

() は付けないことが多いかなと思いますー

Reply:
そのように修正させて頂きました